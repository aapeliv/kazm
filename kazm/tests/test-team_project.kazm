//first class 

class TeamProject {
  int num_members;
  double capacity;
  string name;
};

int main() {
  TeamProject tp;
  tp.num_members = 4;
  tp.capacity = 100.0;
  tp.name = "KAZM";
  print(tp.name);
  print(" has ");
  int_print(tp.num_members);
  print(" members!");
}
