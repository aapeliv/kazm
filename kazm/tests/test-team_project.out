KAZM has 4 members!
